### PR TITLE
deleteMany API changes & tests

### LINKED ISSUE #28: Integrate deleteMany API
Add implementation for deleteMany API

Sample response from the server:

```
{
"status" { "deletedCount" : 100, "moreData" : true}
}
```

When `moreData` is received as `true` from the server, the driver should throw an error back to the client.

### PR DESCRIPTION
**Pull request by kathirsvn**
**What this PR does**:
- Integrated & tested `deleteMany` API
- Updated README for the same

**Which issue(s) this PR fixes**:
Fixes #28 

**Checklist**
- [X] Changes manually tested
- [X] Automated Tests added/updated
- [X] Documentation added/updated
- [X] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: src/collections/collection.ts [hunk 1]
```diff
@@ -255,3 +267,11 @@ export class Collection {
   }
 
 }
+
+export class StargateMongooseError extends Error {
+  message: string;
+  constructor(message: any) {    
+    super(`Operation failed with the following error: ${message}`);
```
Comment (vkarpov15) [inline src/collections/collection.ts]: Same comments as #53. Would be great to report that this was a `deleteMany` that failed, and add a property that contains the command that was sent to the server.

**Review by vkarpov15 — APPROVED** [review]
One quick comment, otherwise LGTM

File: src/collections/collection.ts (hunk 1 above)
Comment (kathirsvn) [reply src/collections/collection.ts]: Makes sense, ll add.

Comment (kathirsvn) [reply src/collections/collection.ts]: Improved this in this [commit](https://github.com/stargate/stargate-mongoose/pull/54/commits/542feceb9014be3b6abeb607e8c5d25c56dc837d).

**Pull request merged**